raft/README.md

```diff
@@ -12,17 +12,14 @@ Below we describe the details to set this up.
     bin/test-raft-server-start.sh config/raft.properties
 
 ### Run Multi Node Quorum ###
-Create 3 separate raft quorum properties as the following
```

Comment:
Do we need to define `process.roles` here?

Reply:
Yeah, you're right. That's pretty annoying. Let me see if I can do anything about it.